Sources/AsyncHTTPClient/HTTPClient+HTTPCookie.swift

```diff
@@ -15,9 +15,9 @@
 import Foundation
 import NIOHTTP1
 
-extension HTTPClient {
+public extension HTTPClient {
```

Comment:
please don't use any qualifiers before `extension`, that's extremely dangerous because it often makes public API by accident.